### DISCUSSION TITLE
How to use img in assets directory when use vue framework?

### DISCUSSION THREAD

**Discussion by chenbimo** [wxt-dev/wxt#180] [Q&A]
![image](https://github.com/wxt-dev/wxt/assets/33003566/f9001419-f9e6-401d-a4c4-75bc2c1b103d)

![image](https://github.com/wxt-dev/wxt/assets/33003566/c2e4eefc-70c2-457f-a425-99687262a2ff)

**Comment by chenbimo** [comment]
Ok!I am careless,so sorry.

`import account_circel from '~/assets/account_circle.svg'`

It's OK!

**Reply by aklinker1** [reply]
Yup, assets behave just like they do in a regular Vite project. Just import them by relative paths or by the path alias.
